specifications/sep-payment-channel-mechanism.md

```diff
@@ -248,33 +234,31 @@ coordinating with each other.
 
 The transactions are constructed as follows:
 
-- C_i, the _closing transaction_, disburses funds to R and changes the signing
-weights on E such that I unilaterally controls E.  C_i has source account E,
-sequence number s_i+1, a `minSeqAge` of O (the observation period time
-duration), and a `minSeqLedgerGap` of O (the observation period ledger count).
+- C_i, the _closing transaction_, disburses funds from EI to ER and/or from ER
+to EI, and changes the signing weights on EI such that I unilaterally controls
+EI, and the signing weights on ER such that R unilaterally controls ER.  C_i has
+source account EI, sequence number s_i+1, a `minSeqAge` of O (the observation
+period time duration), and a `minSeqLedgerGap` of O (the observation period
+ledger count).
 
   The `minSeqAge` and `minSeqLedgerGap` prevents a misbehaving party from
   executing C_i when the channel state has already progressed to a later
   iteration number, as the other party has the period O to invalidate C_i by
   submitting D_i' for some i' > i.
   
   C_i contains operations:
-  - One `BEGIN_SPONSORING_FUTURE_RESERVES` operation that specifies reserve
-  account V as a sponsor of future reserves.
-  - One `CREATE_CLAIMABLE_BALANCE` operation for each trustline that is
-  disbursing funds to R.
-  - One `END_SPONSORING_FUTURE_RESERVES` operation that confirms reserve
-  account V's sponsorship.
-  - One or more `SET_OPTIONS` operation adjusting escrow account E's
-  thresholds to give I full control of E, and removing R's signers.
-  - One or more `SET_OPTIONS` operation adjusting reserve account E's
-  thresholds to give R full control of V, and removing I's signers.
+  - One `PAYMENT` operation for each trustline that is disbursing funds from EI
```

Comment:
This would be limited to 100 payments, and a payment channel should be used for much more than 100 payments so I don't think there's significant value. Also either party can keep copies of all signed txns as proof of each individual payment too.

I think someone could do this if they wanted, but it's probably not recommended.